njoy_processor/Utils_Combiner.py

```diff
@@ -180,20 +196,16 @@ def BuildCombinedData(raw_njoy_data, plot = False, verbose = False):
         rx_name = "(n,{:01d}n)".format(nn)
         n_to_n_nxn_keys.append(rx_name)
     # Freegas thermal scattering
-    n_to_n_freegas_keys = ["mt221"]
+    n_to_n_freegas_keys = ["free-gas"]
     # S(alpha,beta) thermal scattering keys
     n_to_n_sab_inel_keys = ["mt222", "mt229",
                             "mt225", "mt235"]
     n_to_n_sab_el_keys = ["mt230", "mt226", "mt236"]
-    # Neutron to neutron other reactions
-    n_to_n_other_keys = ["(n,np)", "(n,nd)", "(n,na)",
-                         "(n,n2a)", "(n,n3a)"]
     # Neutron to gamma reactions
-    n_to_g_transfer_keys = ["(n,g)", "(n,inel)",
-                            "(n,np)", "(n,nd)",
-                            "(n,p)", "(n,d)", "(n,t)", "(n,a)", "(n,2a)",  
-                            "(n,na)", "(n,n2a)", "(n,n3a)",
-                            "(n,nc)"]
+    n_to_g_transfer_keys = ["(n,g)", "(n,inel)", "(n,np)",
```

Comment:
you are reverting to an old version of the file. do not do that. I will not be able to allow that merge request.

Reply:
you are missing n_to_n_other_keys which is currently in my file. are you sure your branch is based on the latest version of my repo? you have have to do a pull from upstream (google this) to incorporate my changes. A fetch does get my changes in your local database but does not apply them. Maybe fetch from upstream + pull is what you need.